recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -115,10 +115,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "redis++")
-        self.cpp_info.set_property("cmake_target_name", "redis++")
-        self.cpp_info.set_property("pkg_config_name", "redis++")
-        self.cpp_info.components["redis++lib"].set_property("cmake_find_package", "redis++" + "_static" if not self.options.shared else "")
-        self.cpp_info.components["redis++lib"].set_property("cmake_target_name", "redis++" + "_static" if not self.options.shared else "")
```

Comment:
@czoido Then, we should keep this one as well shouldn't we? Given this comment https://github.com/conan-io/conan-center-index/pull/8288/files#r772324693

Reply:
Yes, we have to put it back 👍